l10n_es_account_asset/models/account_asset.py

```diff
@@ -53,6 +53,7 @@ def _inverse_method_percentage(self):
 
     @api.onchange('profile_id')
     def _onchange_profile_id(self):
+        super(AccountAssetAsset, self)._onchange_profile_id()
```

Comment:
En python3 no son necesarios los argumentos de super

Reply:
¡Gracias por el comentario! Me gusta ponerlos por aquello de _Explicit is better than implicit_. Supongo que es cuestión de gustos :)

Reply:
Si es cuestión de gustos, ya sabes que el Python Zen a veces es contradictorio "Simple is better than complex."

Reply:
Si queremos ser puristas (y ya que estamos tocando esto para evitar futuros errores), debemos recoger el resultado de llamar a super y devolverlo al final, por si se pasa por ahí algún cambio de dominio u otro valor. Además, el zen de Python de explícito mejor que ímplicito precisamente se cumple al quitar los argumentos, ya que ahí explícitamente estás diciendo que utilice todo "tal como viene".

```suggestion
        res = super()._onchange_profile_id()
```

y luego al final

```python
return res
```

Reply:
Perfecto, @pedrobaeza. Hago los cambios.